python_ta/__init__.py

```diff
@@ -358,13 +368,15 @@ def get_file_paths(rel_path: AnyStr) -> Generator[AnyStr, None, None]:
                 yield os.path.join(root, filename)  # Format path, from root.
 
 
-def _verify_pre_check(filepath: AnyStr) -> bool:
+def _verify_pre_check(filepath: AnyStr, allowed_pylint: bool) -> bool:
```

Comment:
A better parameter name is `allow_pylint_comments`, matching the config option spelling.

Also, please make sure to *document this new argument in the function docstring*.